Lab1.ipynb

```diff
@@ -289,66 +358,197 @@
    "metadata": {},
    "source": [
     "#### E.\n",
-    "By taking the norm.ppf of our found cdf value, I found that this is equivalent to a 1.386 Sigma in a normal distributinon."
+    "By taking the norm.ppf of our found cdf value, I found that this is equivalent to a ~3 Sigma in a normal distribution."
    ]
   },
   {
    "cell_type": "code",
-   "execution_count": 82,
+   "execution_count": 96,
    "id": "93f73e78-6182-43ce-9f8f-1fd872f86537",
    "metadata": {},
    "outputs": [
     {
      "data": {
       "text/plain": [
-       "1.386294361119891"
+       "2.9957322735539997"
       ]
      },
-     "execution_count": 82,
+     "execution_count": 96,
      "metadata": {},
      "output_type": "execute_result"
     }
    ],
    "source": [
-    "stats.norm.ppf(stats.lognorm.cdf(0.4,s,loc=0,scale = 0.1))"
+    "stats.norm.ppf(stats.lognorm.cdf(my_measurement,s,loc=0,scale = 0.1))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "18855c96-d91b-4c83-a9cf-83a944584a84",
+   "metadata": {},
+   "source": [
+    "In order to achieve 5-Sigma for this distribution, I would need to find a measurement of lognorm.cdf = norm.cdf(5). To find this I used the inverse survival function, and found that I would need a measurement of 14.84 for it to be a 5-Sigma result."
    ]
   },
   {
    "cell_type": "code",
-   "execution_count": null,
+   "execution_count": 105,
    "id": "1380a57d-4c43-4e7a-bbba-eb769be6af02",
    "metadata": {},
-   "outputs": [],
-   "source": []
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "14.841315909815032"
+      ]
+     },
+     "execution_count": 105,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "stats.lognorm.isf(1-stats.norm.cdf(5),s,loc=0,scale = 0.1)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "1e38093a-8fc8-4166-b845-d42dfc4a1594",
+   "metadata": {},
+   "source": [
+    "## Non-continuous distributions (Poisson)\n",
+    "\n",
+    "#### A.\n",
+    "The Poisson distribution is a model for when the average time between events is known, but when the events will occur is not known. For a given number of events k, the probability of measuring k events in a given timeframe is:\n",
+    "\n",
+    "$$Pr(x=k) = \\frac{\\lambda^k e^{-\\lambda}}{k!} $$ \n",
+    "\n",
+    "where $\\lambda$ is the expected value of x, and its variance. This distribution is discrete, because you cannot measure a fractional number of events taking place.\n",
+    "\n",
+    "#### B."
```

Comment:
For this section, you did a great job of creating plots of different parameters, but you should also explain the features of each plot in words. How does changing the parameters of your Poisson distribution affect its shape?